osf/migrations/0118_merge_20180704_1251.py

```diff
@@ -0,0 +1,16 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.13 on 2018-07-04 17:51
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
```

Comment:
Generally speaking, merge migrations shouldn't be required in PRs, unless there's an upstream conflict -- is this necessary?